tol_lab_share/message_properties/definitions/labware_type.py

```diff
@@ -44,24 +44,24 @@ def pad_number(self, number: int) -> str:
             padded_number = f"0{padded_number}"
```

Comment:
if padded_number is 10, this will make 010 . Is this correct?

Reply:
No because the line before it checks for a length `1` string, but I don't know why Eduardo implemented it this way when `str` has a `zfill()` method precisely for padding numbers with zeroes.  I'll update it.